storey/flow.py

```diff
@@ -629,3 +664,150 @@ def build_flow(steps):
             cur_step.to(next_step)
             cur_step = next_step
     return steps[0]
+
+
+class V3ioTable(NeedsV3ioAccess):
+    def __init__(self, table, webapi=None, access_key=None):
+        NeedsV3ioAccess.__init__(self, webapi, access_key)
+        self.table = table
+
+    async def save_schema(self, schema):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+
+        try:
+            response = await client_session.put(f'{self._webapi_url}/{self.table}/{schema_file_name}',
+                                                headers=self._get_put_file_headers, data=json.dumps(schema), ssl=False)
+            if not response.status == 200:
+                body = await response.text()
+                raise V3ioError(f'Failed to save schema file. Response status code was {response.status}: {body}')
+        except BaseException as ex:
+            raise ex
+        finally:
+            await client_session.close()
+
+    async def load_schema(self):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
```

Comment:
Why start a new session for every operation? Opening and closing a connection every time will slow these down considerably.

Reply:
This is really true everywhere. We need to keep the session in the class.